libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -128,6 +140,95 @@ describe("Drilling", () => {
                 .should("have.text", "Column chart with years › 2010 › Q2/2010");
             drillModal.getChart().hasDataLabelValues(valueAttribute);
         });
+
+        it("Drilling down continue on a table ", () => {
+            Navigation.visit("dashboard/dashboard-table-drill-down");
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN, YEAR_CLOSE);
+            assertTableValues.forEach((value, index) => {
+                firstWidget.waitTableLoaded().getTable().hasCellValue(index, 0, value);
+            });
+
+            firstWidget.waitTableLoaded().getTable().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should("have.text", `table has row column measure › ` + `${assertTableValues[0]}`);
+            assertDrillToProduct.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `table has row column measure › ` +
+                        `${assertTableValues[0]}` +
+                        " › " +
+                        `${assertDrillToProduct[0]}`,
+                );
+            assertDrillToYear.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.back();
+            drillModal.getTable().waitLoaded().click(1, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `table has row column measure › ` +
+                        `${assertTableValues[0]}` +
+                        " › " +
+                        `${assertDrillToProduct[1]}`,
+                );
+            drillModal.close();
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN);
+        });
+
+        it("Drilling down on a table that has two drillable attributes separate", () => {
+            const assertDrillToQuarterValue = ["Q2/2010"];
+
+            Navigation.visit("dashboard/implicit-drill");
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN, YEAR_CLOSE);
+            firstWidget.waitTableLoaded().getTable().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should("have.text", `Table with years › ` + `${assertDrillToYear[0]}`);
+            drillModal.getTable().waitLoaded().click(0, 0);
```

Comment:
why do you click again on table. at here, we can verify some data in table after click on line 191